client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -57,6 +70,22 @@
      */
     private final List<ExtraHeuristicsDto> extras = new CopyOnWriteArrayList<>();
 
+    /**
+     * a map of interface schemas for RPC service under test
+     */
+    private final Map<String, InterfaceSchema> rpcInterfaceSchema = new HashMap<>();
```

Comment:
to avoid issues with non-determinism, prefer `LinkedHashMap` over `HashMap`

Reply:
updated